templates/publisher/account-snaps.html

```diff
@@ -169,18 +179,32 @@ <h3 class="p-heading--five">Join the community</h3>
   {% if registered_snaps %}
     <section class="p-strip is-shallow u-no-padding--top">
       <div class="row">
-        <div class="col-12">
-          <hr />
-          <h4>My registered snap names</h4>
-        </div>
+        <h4 class="p-heading--five">Registered names ({{ registered_snaps|count }})</h4>
       </div>
       <div class="row">
-        <p>
-          Names you own but have not pushed a snap to:
-        </p>
-        <p>
-          {% for registered_snap in registered_snaps|sort %}{% if loop.index > 1 %}, {% endif %}{{ registered_snap }}{% endfor %}
-        </p>
+        <table>
+          <tbody>
+            {% for snap in registered_snaps|sort %}
+              {% with dispute_pending = registered_snaps[snap].status == "DisputePending" %}
+                <tr class="p-snapcraft-dispute-list__item">
+                  <td width="25%">
+                    {{ snap }}
+                    {% if dispute_pending %}
+                      &nbsp;<i class="p-icon--warning p-snapcraft-dispute-list__icon"></i>
+                    {% endif %}
+                  </td>
+                  <td>
+                    {% if dispute_pending %}
+                      <span class="p-snapcraft-dispute-list__muted">(Snap name disputed)</span>
```

Comment:
I think (snap name disputed) makes it sound like the dispute is not pending anymore. I would maybe use (snap name in dispute). Thoughts @therealjuan @Lukewh?

Reply:
Name dispute in progress?

Reply:
Updated :+1:

Reply:
:+1: